src/app/desktop/components/Instances/Instance.js

```diff
@@ -164,6 +164,44 @@ const Instance = ({ instanceName }) => {
 
   const isPlaying = startedInstances[instanceName];
 
+  const calcTime = minutes => {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['calcTime' is assigned a value but never used.](https://app.codacy.com/gh/gorilla-devs/GDLauncher/pullRequest?prid=5468353)

Reply:
How tho? I didn't change that..